Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'http://rubygems.org'
-ruby '2.3.1'
+ruby '2.1.5'
```

Comment:
hmm, I was able to run rails server locally with ruby 2.3.1 and rails 3.0.19. were you having problems?